core/management/commands/maintenance.py

```diff
@@ -0,0 +1,157 @@
+        self.stdout.write("\n\n")
+
+        new_title = raw_input("Provide desired title (ENTER to default): ")
+        if len(new_title) > 0:
+            title = new_title
+
+        new_message = raw_input("Provide descriptive message (ENTER to default): ")
+        if len(new_message) > 0:
+            message = new_message
+
+        return (title, message)
+
+    def _gather_start_date(self):
+        start_date = default_start_date()
+
+        banner = "Default:"
+        self.stdout.write(banner)
+        self.stdout.write("-" * len(banner))
+        self.stdout.write("- start date: {0}".format(start_date))
+        self.stdout.write("\n")
+
+        # ISO 8601 OR GET OUT! https://xkcd.com/1179/
+        new_start_date = raw_input("Provider different start date, format: YYYY-MM-DD HH:MM\n(ENTER to default): ")
+        if len(new_start_date) > 0:
+            try:
+                start_date = timezone.datetime.strptime(
+                    new_start_date, "%Y-%m-%d %I:%M")
+            except ValueError:
+                self.stderr.write("Please use the ISO 8601 format:")
+                self.stderr.write(" - https://xkcd.com/1179/")
+
+        return start_date
+
+
+    def handle_start(self):
+        banner = "\nGathering Maintenance Information ..."
+        self.stdout.write(banner)
+        self.stdout.write("=" * len(banner))
+        self.stdout.write("\n")
+
+        while True:
+            title, message = self._gather_input()
+
+            self.stdout.write("\nTitle: {0}".format(title))
+            self.stdout.write("Message: {0}".format(message))
+            self.stdout.write("\n")
+
+            option = raw_input("Continue? [Y/n] ")
+            self.stdout.write("\n")
+
+            if _continue(option):
+                break
+
+        while True:
+            start_date = self._gather_start_date()
+
+            self.stdout.write(
+                "\n- Start date: {0}\n\n".format(start_date))
+
+            option = raw_input("Continue? [Y/n] ")
+            self.stdout.write("\n")
+
+            if _continue(option):
+                break
+
+        new_record = MaintenanceRecord.objects.create(
+            start_date=default_start_date())
+
+        new_record.title = title
+        new_record.message = message
+        new_record.save()
+
+        self.stdout.write(self.style.SUCCESS("MaintenanceRecord saved ..."))
+
+        return True
+
+    def handle_stop(self):
+        records = MaintenanceRecord.active()
+
+        self.stdout.write(
+            "Preparing to process {0} records ...".format(len(records)))
+
+        for record in records:
+            self.stdout.write(" - End dating ... {0}".format(record))
+
+            record.end_date = timezone.now()
+            record.save()
+
+        self.stdout.write("Done ...")
+
+        return True
+
+    def handle(self, *args, **options):
+        cmd = options['command']
+
+        result = False
+
+        if cmd == 'start':
```

Comment:
nice @lenards  Thanks!

Reply:
the collapse/expand part of the PR has a "<details/>" element with screenshots of example interactions.

Reply:
Look here =) 

<img width="572" alt="details-examples-blah-start-stop" src="https://user-images.githubusercontent.com/5923/30611548-87ff3c02-9d36-11e7-9d14-fe4edfccdc1f.png">

Reply:
nice!! Yeah, totally overlooked it :) Thanks for making this image.